torchvision/models/_utils.py

```diff
@@ -65,3 +67,9 @@ def forward(self, x):
                 out_name = self.return_layers[name]
                 out[out_name] = x
         return out
+
+
+def overwrite_eps(model, eps):
+    for module in model.modules():
+        if isinstance(module, FrozenBatchNorm2d):
+            module.eps = eps
```

Comment:
This is fine here, but I would have moved it instead to `detection/_utils.py` as this fix is specific to detection models so in practice there is no need to do it here.

But that's not a blocking matter, so I'm going to be merging this as is.

Reply:
done in this PR.